script/script.js

```diff
@@ -0,0 +1,17 @@
+ document.querySelector('#create_task').addEventListener('click', myClick);
+
+ function myClick(){
+    //console.log('work')
+    let text = document.querySelector('.i-1').value;
```

Comment:
You are searching by class and your code expects only 1 element to be found.
A class can be assigned to multiple elements and this will break your code.
Make sure to use ID instead of CLASS.

Reply:
Replaced class on id